jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3StreamConnection.java

```diff
@@ -56,6 +56,15 @@ public HTTP3StreamConnection(QuicStreamEndPoint endPoint, Executor executor, Byt
         parser.init(MessageListener::new);
     }
 
+    public void onFailure(Throwable failure)
+    {
+        if (networkBuffer != null)
+        {
+            networkBuffer.release();
+            networkBuffer = null;
+        }
```

Comment:
```suggestion
        releaseInputBuffer();
```

Can you also either change the name of that method or of the field, so they are either both `network` or both `input`.

Reply:
You're right, this buffer should be called `inputBuffer` and the related methods named likewise. Let me clean this.